config/dist/webpack-assets.json

```diff
@@ -0,0 +1,11 @@
+{
+  "vendor": {
```

Comment:
did you really want this generated code committed? or should it go in the gitignore?